docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -235,9 +235,36 @@
     }
 
     h2 {
-      @include hb-heading-3;
+      @include hb-heading-5;
       margin: 0;
       overflow-wrap: break-word;
+
+      // If this h2 is in a grid,
+      // keep it smaller longer.
```

Comment:
Should we also do this with the Vertical Linked Card Title?

Reply:
great call!